extensions/amp-list/0.1/amp-list.js

```diff
@@ -341,18 +398,26 @@ export class AmpList extends AMP.BaseElement {
   /**
    * Schedules a fetch result to be rendered in the near future.
    * @param {!Array|?JsonObject|string|undefined} data
+   * @param {boolean=} opt_append
    * @return {!Promise}
    * @private
    */
-  scheduleRender_(data) {
+  scheduleRender_(data, opt_append = false) {
     dev().info(TAG, 'schedule:', data);
     const deferred = new Deferred();
     const {promise, resolve: resolver, reject: rejecter} = deferred;
+
     // If there's nothing currently being rendered, schedule a render pass.
     if (!this.renderItems_) {
       this.renderPass_.schedule();
     }
-    this.renderItems_ = {data, resolver, rejecter};
+
+    this.renderItems_ = {data, opt_append, resolver, rejecter};
```

Comment:
nitL it's not really `opt` at this point, just `append` would do.

Reply:
Done. Thank you!